doc/Addons.md

```diff
@@ -357,6 +357,13 @@ Hook data:
     'item' => item array (input)
     'html' => converted item body (input/output)
 
+### 'put_item_in_cache'
+Called after prepare_text in put_item_in_cache().
+Hook data:
+    'item' => item array (input)
```

Comment:
The indentation style is not consistent here.

Reply:
Also I'm not sure if there should be an empty line between the text and the listing of the data in Markdown.